src/screens/APIUtils/APIUtils.res

```diff
@@ -301,6 +301,15 @@ let useGetURL = () => {
         }
       }
 
+    /* USER MANGEMENT REVAMP */
+    | USER_MANAGEMENT_V2 => {
+        let userUrl = `user`
+        switch userRoleTypes {
+        | USER_LIST => `${userUrl}/user/v2/list`
+        | _ => ""
+        }
+      }
```

Comment:
```suggestion
    | USER_MANAGEMENT_V2 = `${user}/user/v2/list`
```
 If it's single URL then why not to use this way and remove when revamp is done. Just a suggestion.

Reply:
More APIs are about to come for user_management .